detect_variants/detect_variants.cwl

```diff
@@ -25,6 +25,10 @@ inputs:
         secondaryFiles: [.tbi]
     strelka_exome_mode:
         type: boolean
+    mutect_scatter_count:
+        type: int?
+    mutect_artifact_detection_mode:
```

Comment:
Yeah... I only added it out of suspicion that the optional parameters weren't working properly.  Now that more things are straightened out I could try making them optional again.